metadata_checks/metadata_checker.py

```diff
@@ -0,0 +1,35 @@
+"""Module for metadata checks."""
```

Comment:
Imho we can group all test functions in a package “checks”. Then we could add a file metadata.py which contains the metadata functions. I don’t think we will have more metadata files to have a separate package called metadata_checks. Wdyt?